### DISCUSSION TITLE
Query Builder Basic View - Add Way To Identify Unmapped Fields

### DISCUSSION THREAD

**Discussion by carlosmbe** [specify/specify7#5882] [Feature Requests]
If for some reason, a field becomes unavailable and it was in saved query. When run, the query will throw a does not exist error.

In detailed view, this isn't really a problem as you can easily identify the problem. 

However, in basic view, there is no mechanism for this. You would have to switch between the two to make the needed changes

This is more of a small quality of life feature request but it would be pretty cool if we added a danger icon or a red star to fields that are unmapped in the basic view

https://github.com/specify/specify7/assets/53784701/2dd11674-c446-445d-8d77-72e911b1bca4

**Comment by maxpatiiuk** [comment]
I think this can happen for tree ranks only at the moment.

There is also a case with you not having permissions to the tables/fields in question, or a pick list no longer existing or a field being marked as hidden, but @CarolineDenis added warnings for those cases
